src/commands/Tools/emotes.ts

```diff
@@ -0,0 +1,83 @@
+import { MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserRichDisplay } from '../../lib/structures/UserRichDisplay';
+import { BrandingColors } from '../../lib/util/constants';
+import { getColor } from '../../lib/util/util';
+
+export default class extends SkyraCommand {
+
+	private kFilter = this.removeNullAndUndefined.bind(this);
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 10,
+			description: language => language.tget('COMMAND_EMOTES_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_EMOTES_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
+			runIn: ['text']
+		});
+	}
+
+	public async run(message: KlasaMessage) {
+		const response = await message.sendEmbed(new MessageEmbed()
+			.setDescription(message.language.tget('SYSTEM_LOADING'))
+			.setColor(BrandingColors.Secondary));
```

Comment:
afaik you're supposed to use `getColor(msg)`

Reply:
For the loading system message it should be brandingcolors. At least that's based on what Kyra made it like way back when I did my first work with urbandictionary soooo.....